wikidata/client.py

```diff
@@ -179,19 +180,28 @@ def decode_datavalue(self,
 
     def request(self, path: str) -> Union[
         bool, int, float, str,
-        Mapping[
-            str,
-            Union[bool, int, float, str, Mapping[str, object], Sequence]
-        ],
-        Sequence[Union[bool, int, float, str, Mapping[str, object], Sequence]]
+        Mapping[str, Union[bool, int, float, str,
+                           Mapping[str, object], Sequence]],
+        Sequence[Union[bool, int, float, str, Mapping[str, object], Sequence]],
```

Comment:
Is this style change functionally necessary?  Or does lint require this change?

Reply:
Linting otherwise line is too long

Reply:
Fair enough.